### ISSUE TITLE
Documentation page doesn't exists

### ISSUE THREAD

**Issue by jorgefuertes** [ericelliott/h5Validate#101]
"For demo and usage, see the h5Validate homepage" links to http://ericleads.com/h5validate.
This page doesn't exists.

Thanks.

**Comment by ericelliott** [comment]
Yep. Pull request welcome.

**Comment by zenopopovici** [comment]
A direct link: https://web.archive.org/web/20150319095922/http://ericleads.com/h5validate/

Will find the time to rewrite it.

**Comment by julmot** [comment]
What a bad user experience.

**Comment by ericelliott** [comment]
@julmot It's open source. Feel free to contribute a pull request. I haven't used this library myself in several years. I leave it here for other people to benefit from.

**Comment by ericelliott** [comment]
I've updated the project status:

> A lot has changed since I wrote this library in 2010. It has been several years since I used the library myself. I leave it here because other people might find it useful. There are some open issues, and I have no intention of updating or maintaining this library myself. Pull requests are welcome.

> If you'd like to become the new maintainer of this project, open an issue and we'll discuss it.

**Comment by zenopopovici** [comment]
My team thought of cleaning it up for jQuery 3.0. It's still the best validation library for HTML5 out-there.

**Comment by julmot** [comment]
>  It's still the best validation library for HTML5 out-there.

I can't agree with that. What others have you tried?

**Comment by zenopopovici** [comment]
@julmot I don't think this is the place to discuss this. I was replying to @ericelliott. Feel free to contribute if you wish.

**Comment by julmot** [comment]
@zenopopovici I think this is the best place to discuss this. @ericelliott doesn't maintain this project since Feb 15, 2014 and searches for a new maintainer. I think it will be hard to find one if the arguments for this plugin aren't transparent. Saying `It's still the best validation library for HTML5 out-there.` without adding arguments is kinda implausible.

**Comment by zenopopovici** [comment]
@julmot I'm not getting dragged into this. It's a waste of everybody's time :)

**Comment by julmot** [comment]
@zenopopovici 

> I'm not getting dragged into this. It's a waste of everybody's time

:laughing: If it's a waste of time saying what makes this plugin uniquely, it's a waste of time finding a maintainer and making the plugin available. :beers:

**Comment by ericelliott** [comment]
@julmot Are there better jQuery validation plugins out there? Feel free to post links for people who are looking for maintained alternatives.

**Comment by zenopopovici** [comment]
@ericelliott Here you go: https://github.com/ericelliott/h5Validate/pull/102

**Comment by julmot** [comment]
@ericelliott 

- https://jqueryvalidation.org/
- http://formvalidation.io
- http://www.formvalidator.net/
- http://parsleyjs.org/
- http://jaymorrow.github.com/validatr/
- https://github.com/hsnaydd/validetta
- https://github.com/whatgoodisaroad/validity
- http://1000hz.github.io/bootstrap-validator/
- https://github.com/alfredobarron/smoke

**Comment by ericelliott** [comment]
I guess better is in the eye of the beholder.
